src/main/java/org/folio/marc/migrations/domain/entities/ChunkStep.java

```diff
@@ -0,0 +1,67 @@
+package org.folio.marc.migrations.domain.entities;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.sql.Timestamp;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.folio.marc.migrations.domain.entities.types.OperationStep;
+import org.folio.marc.migrations.domain.entities.types.StepStatus;
+import org.hibernate.annotations.JdbcTypeCode;
+import org.hibernate.type.SqlTypes;
+
+@Getter
+@Setter
+@Entity
+@ToString
+@Builder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "operation_chunk_step")
+public class ChunkStep {
+
+  @Id
+  @Column(nullable = false)
+  private UUID id;
+
+  @Column(nullable = false)
+  private UUID operationId;
+
+  @Column(nullable = false)
+  private UUID operationChunkId;
+
+  @Enumerated(EnumType.STRING)
+  @JdbcTypeCode(SqlTypes.NAMED_ENUM)
+  @Column(nullable = false)
+  private OperationStep operationStep;
+
+  @Column
+  private String entityErrorChunkFileName;
+
+  @Column
+  private String errorChunkFileName;
+
+  @Enumerated(EnumType.STRING)
+  @JdbcTypeCode(SqlTypes.NAMED_ENUM)
+  @Column(nullable = false)
+  private StepStatus status;
+
+  @Column(name = "start_time_mapping")
+  private Timestamp stepStartTime;
+
+  @Column(name = "end_time_mapping")
+  private Timestamp stepEndTime;
```

Comment:
Why do there start_time_mapping and end_time_mapping, can't it be step for saving (based on operationstep enum)

Reply:
That's a mistake, it's different in schema. It didn't break anything because not used with JPA. Will remove